package.json

```diff
@@ -63,5 +63,6 @@
       "npm run prettier-ts -- --write",
       "git add"
     ]
-  }
+  },
+  "prettier": {}
```

Comment:
People may have their editor configured with default prettier settings. Explicitly setting an empty config ensures that the default settings are used instead of those set by the user.